File: year-2/programming-design/c#/motorcycle-contest/MotorcycleContest/Domain/Entities/Team.cs
using System;

namespace MotorcycleContest.Domain.Entities
{
    public class Team : Entity<int>
    {
        public String Name { get; set; }

        public Team()
        {
            this.Name = null;
        }

        public Team(String name)
        {
            this.Name = name;
        }

        public override string ToString()
        {
            return "Team " + ID + ", name " + Name + "; ";
        }
    }
}

File: year2/mpp/java/MotorcycleGrpcClient/MotorcycleCommon/src/main/java/domain/Team.java
package domain;

public class Team extends Entity<Integer> {
    private String name;

    public Team() {
        this.name = null;
    }

    public Team(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @Override
    public String toString() {
        return "Team " + getId() + ", name " + name + "; ";
    }
}
